tensorflow/core/common_runtime/eager/mkl_eager_op_rewrite.cc

```diff
@@ -69,49 +61,51 @@ class MklEagerOpRewrite : public EagerOpRewrite {
 
   // Calls op-specific rewrite function to create new MKL op.
   Status RewriteToMklOp(EagerOperation* orig_op,
-                        std::unique_ptr<EagerOperation>* mkl_op,
-                        const int op_idx);
+                        std::unique_ptr<EagerOperation>* mkl_op);
 
-  // Checks whether we can rewrite the op to MKL one or not.
-  bool ShouldRewriteOp(EagerOperation* op, int* op_idx);
+  // Check whether we can rewrite the op to MKL one or not.
+  bool ShouldRewriteOp(EagerOperation* op);
 
   // Default rewrite rule to be used when rewrite should happen without any
   // restriction.
   static bool AlwaysRewrite(EagerOperation* op) { return true; }
 
-  // Checks if kernel is registered for a particular op.
-  bool FastCheckIfKernelRegistered(string op_name, DataType dt);
+  // Check if kernel is registered for a particular op.
+  bool FastCheckIfKernelRegistered(std::string op_name, DataType dt);
+
+  // This is called by FastCheckIfKernelRegistered once per unique op name
+  // and data type.
+  bool SlowCheckIfKernelRegistered(std::string op_name, DataType dt);
 
-  // This is called by FastCheckIfKernelRegistered once per unique op name and
-  // data type.
-  bool SlowCheckIfKernelRegistered(string op_name, DataType dt);
+  // Helper function to insert mkl_eager_ops to Map
+  void InsertMKLEagerOps(MklEagerOp op);
 
-  // map used by FastCheckIfKernelRegistered.
-  std::unordered_map<string, bool> registered_kernels_map;
+  // Map used by FastCheckIfKernelRegistered.
+  std::unordered_map<std::string, bool> registered_kernels_map;
```

Comment:
Nit: Class member naming.
```suggestion
  std::unordered_map<std::string, bool> registered_kernels_map_;
```
Sorry I didn't notice this in the last PR.

Reply:
no worries. My bad!